app-editors/jedit/jedit-5.4.0-r1.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+JAVA_PKG_IUSE="doc test"
+inherit desktop java-pkg-2 java-ant-2 xdg-utils
+
+DESCRIPTION="Programmer's editor written in Java"
+HOMEPAGE="http://www.jedit.org"
+SRC_URI="mirror://sourceforge/${PN}/${PN}${PV}source.tar.bz2"
+
+LICENSE="BSD GPL-2"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+SLOT="0"
+
+CP_DEPEND="dev-java/jsr305:0"
+
+# Restricting to java 1.8 as it fails to compile with newer java versions
```

Comment:
could you please be more specific?

Reply:
> could you please be more specific?

Could you recommend, is there another way of solving https://bugs.gentoo.org/828559?